petastorm/tests/test_pytorch_dataloader.py

```diff
@@ -230,12 +240,15 @@ def test_mem_cache_num_epochs_without_mem_cache_error(two_columns_non_petastorm_
 @pytest.mark.parametrize('shuffling_queue_capacity', [20, 0])
 @pytest.mark.parametrize('reader_factory', [make_batch_reader, make_reader])
 @pytest.mark.parametrize('num_epochs', [1, 2, 3, None])
+@pytest.mark.parametrize('async_batch_queue_capacity', [0, 2, 4])
```

Comment:
The test is launched 2x2x4x3=48 times. Let's consolidate the parametrization to actually interesting cases and reduce test run time:
```python
@pytest.mark.parametrize('shuffling_queue_capacity,reader_factory,num_epochs,async_batch_queue_capacity', [[set1], [set2],...]
```